notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,620 @@
+{
```

Comment:
<p>Not quite true about <code>object</code>, especially if https://github.com/pandas-dev/pandas/pull/27949 is merged for 1.0.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Even when that is merged, it will not yet be the default, right? </p><p> </p><p>I would reword to somethinglike "any python object, but in practice often means strings"</p>

Reply:
<p>As pointed in https://github.com/pandas-dev/pandas/issues/26831#issuecomment-529716754 we will exclude the discussion on object types in the first tutorial.</p>